tests/test_back.js

```diff
@@ -123,58 +122,49 @@ test('nockBack returns a promise when neither options nor nockbackFn are specifi
 })
 
 test('nockBack throws an exception when a hook is not a function', t => {
-  nockBack.fixtures = `${__dirname}/fixtures`
   nockBack.setMode('dryrun')
-  setOriginalModeOnEnd(t, nockBack)
-
   t.throws(
     () => nockBack('goodRequest.json', { before: 'not-a-function-innit' }),
     { message: 'processing hooks must be a function' }
   )
-
   t.end()
 })
 
 test('nockBack.setMode throws an exception on unknown mode', t => {
   t.throws(() => nockBack.setMode('bogus'), { message: 'Unknown mode: bogus' })
-
   t.end()
 })
 
 test('nockBack returns a promise when nockbackFn is not specified', t => {
-  nockBack.fixtures = `${__dirname}/fixtures`
-
   nockBack('test-promise-fixture.json', { test: 'options' }).then(params => {
     t.type(params.nockDone, 'function')
     t.type(params.context, 'object')
     t.end()
   })
 })
 
-test('nockBack wild tests', nw => {
-  // Manually disable net connectivity to confirm that dryrun enables it.
-  nock.disableNetConnect()
+test('with wild, normal nocks work', t => testNock(t))
 
-  nockBack.fixtures = `${__dirname}/fixtures`
+test('wild enables net connect', t => {
+  nock.disableNetConnect()
   nockBack.setMode('wild')
-  setOriginalModeOnEnd(nw, nockBack)
-
-  nw.test('normal nocks work', t => testNock(t))
-
-  nw.test("nock back doesn't do anything", t =>
-    nockBackWithFixtureLocalhost(t, false)
-  )
-
-  nw.end()
+  // TODO: It would be nice if there were a cleaner way to assert that net
+  // connect is allowed.
+  nockBackWithFixtureLocalhost(t)
+  t.end()
 })
 
-test('nockBack dryrun tests', nw => {
-  // Manually disable net connectivity to confirm that dryrun enables it.
-  nock.disableNetConnect()
+test("with wild, nock back doesn't do anything", t =>
+  nockBackWithFixtureLocalhost(t)
+)
 
-  nockBack.fixtures = `${__dirname}/fixtures`
-  nockBack.setMode('dryrun')
-  setOriginalModeOnEnd(nw, nockBack)
+test('nockBack dryrun tests', nw => {
+  nw.beforeEach(done => {
```

Comment:
Since we now reset the `nockBack` state before each test, a `beforeEach` hook is needed here.